packages/server/graphql/intranetSchema/queries/helpers/authCountByDomain.ts

```diff
@@ -19,13 +19,15 @@ const authCountByDomain = async (
         `SELECT count(*)::float as total, split_part(email, '@', 2) as "domain" from "User"
          WHERE (NOT $1 OR inactive = FALSE)
          AND "${filterField}" >= $2
-         GROUP BY split_part(email, '@', 2)`,
+         GROUP BY split_part(email, '@', 2)
+         ORDER BY total DESC`,
         [countOnlyActive ?? false, after]
       )
     : await pg.query<DomainTotal>(
         `SELECT count(*)::float as total, split_part(email, '@', 2) as "domain" from "User"
          WHERE (NOT $1 OR inactive = FALSE)
-         GROUP BY split_part(email, '@', 2)`,
+         GROUP BY split_part(email, '@', 2)
+         ORDER BY total DESC`,
```

Comment:
Just checked and it was ordered when read from rethink https://github.com/ParabolInc/parabol/pull/5408/files#r753975310